### DISCUSSION TITLE
How does casting a spell work?

### DISCUSSION THREAD

**Discussion by mifriis** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#71] [Q&A] [answered]
We are transitioning from tabletop simulator and regular pen&paper to foundry. 
I have a spell caster, and i can't quite figure out how the spell system works.

Say i cast Stonestorm using 2 WP. I then add two "base" and press roll. Power level will be 2+Sixes which is correctly laid out.

Say i cast Stonestorm with some small rocks as ingridient, how do i configure that? Do i add an additional base and a guaranteed success?

Say i cast Stonestorm, using 2 WP, have it written down in my grimoir, using an ingridient, and safecasting it. I have no idea 🗡️ 

Can you help me understand base and guaranteed success?

**Comment by aMediocreDad** [comment, marked as answer]
Hey there!

I can understand this is a bit confusing. The system does not support adding these things in the dice roller. I have not reworked this part of the system yet, but I am working on adding functionality to the diceroller at the moment so this is definitely something I can look at. 

> Say i cast Stone storm with some small rocks as ingredient, how do i configure that? Do i add an additional base and a guaranteed success?

Here you track the added power level from the ingredient separately. Auto Successes simply means you roll a 6 on the amount of dice, and assumes you spent the WP equal to the baseDice. An ingredient does not add a die, it simply increases the power level of the spell.

> Say i cast Stone storm, using 2 WP, have it written down in my grimoire, using an ingredient, and safe casting it. I have no idea 🗡️

Like above you track the ingredient separately. Safecasting means you roll one less dice but still spend the WP so you track one more "power level" separately. So we are now at 2+ whatever is rolled by the dice roller.

In other words, the way it was originally designed the dice roller is only there to determine what happens to the dice you roll as a result of casting the spell. It does NOT give you a final total power level of your spell. You add power levels gained from ingredients and safe casting to the power level rolled.

Is that a clear answer?

**Reply by mifriis** [reply]
That's great, thank you. 

Sometimes it just helps to think of this not as a computergame where the output is the truth, but rather as dice rolls that need to be interpreted, similiar to what we do at the table. 

Amazing job on the whole module, it will bring us lots of fun. Crypt of the Mellified Mage and The Spire of Quetzel are on my to-buy-list when/if they get foundryfied too :)

**Reply by aMediocreDad** [reply]
Super! Please go ahead and mark it as answer:) If you have discord I would recommend checking out the foundry vtt discord server and ask questions in the free-league channel:+1:
